pims/imageio_reader.py

```diff
@@ -39,6 +46,23 @@ def __init__(self, filename, **kwargs):
         self._shape = first_frame.shape
         self._dtype = first_frame.dtype
 
+        self._setup_axes()
+        self._register_get_frame(self.get_frame, 'tyx')
```

Comment:
I think it should be `'yx'` here as `get_frame` returns a frame with only y, x axes.